frontend/src/app/core/services/training.service.ts

```diff
@@ -244,4 +245,16 @@ export class TrainingService {
       confirmUploadRequestBody,
     );
   }
+
+  public deleteCertificates(
+    workplaceUid: string,
+    workerUid: string,
+    trainingUid: string,
+    filesToDelete: Array<any>,
```

Comment:
```suggestion
    filesToDelete: TrainingCertificate[],
```
from the usage, seems that `filesToDelete` have this type